<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Test</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icone.qrc">
    <normaloff>:/iconeWindows/icone/usb.ico</normaloff>:/iconeWindows/icone/usb.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_command">
          <property name="text">
           <string>Command</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_cmd"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_byte">
          <property name="text">
           <string>Byte reception</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_byte">
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="button_Send">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_console">
        <property name="text">
         <string>Show Console :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Console" name="console_Window">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClearConsole"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeSerial/icone/usb.png</normaloff>:/iconeSerial/icone/usb.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeSerial/icone/disconnect.png</normaloff>:/iconeSerial/icone/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeWindows/icone/remove.png</normaloff>:/iconeWindows/icone/remove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeSerial/icone/settings.png</normaloff>:/iconeSerial/icone/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeWindows/icone/question.png</normaloff>:/iconeWindows/icone/question.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeWindows/icone/Qt_logo_2016.svg.png</normaloff>:/iconeWindows/icone/Qt_logo_2016.svg.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionClearConsole">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeWindows/icone/broom.png</normaloff>:/iconeWindows/icone/broom.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Console</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">console.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icone.qrc"/>
 </resources>
 <connections/>
</ui>
